Kitodo/src/main/java/org/kitodo/production/services/data/PropertyService.java

```diff
@@ -198,25 +148,8 @@ private QueryBuilder getQueryForType(String type) {
      *            true or false
      * @return list of JSON objects with batches of exact type
      */
-    List<Map<String, Object>> findByTitleAndValue(String title, String value, String type, boolean contains)
-            throws DataException {
-        BoolQueryBuilder query = new BoolQueryBuilder();
-        query.must(createSimpleQuery(PropertyTypeField.TITLE.getKey(), title, contains, Operator.AND));
-        query.must(createSimpleQuery(PropertyTypeField.VALUE.getKey(), value, contains, Operator.AND));
-        if (Objects.nonNull(type)) {
-            query.must(createSimpleQuery(PropertyTypeField.TYPE.getKey(), type, true));
-        }
-        return findDocuments(query);
-    }
-
-    @Override
-    public PropertyDTO convertJSONObjectToDTO(Map<String, Object> jsonObject, boolean related) throws DataException {
-        PropertyDTO propertyDTO = new PropertyDTO();
-        propertyDTO.setId(getIdFromJSONObject(jsonObject));
-        propertyDTO.setTitle(PropertyTypeField.TITLE.getStringValue(jsonObject));
-        propertyDTO.setValue(PropertyTypeField.VALUE.getStringValue(jsonObject));
-        propertyDTO.setCreationDate(PropertyTypeField.CREATION_DATE.getStringValue(jsonObject));
-        return propertyDTO;
+    List<Property> findByTitleAndValue(String title, String value, String type, boolean contains) {
+        return getByQuery("from Property as property where property.title='" + title + "' and property.value='" + value + "'");
```

Comment:
Please use `getByQuery` with second parameter with list of to map values. If not this is introducing a possible SQL injection.